mm2src/coins/nft/storage/sql_storage.rs

```diff
@@ -123,6 +153,20 @@ fn get_nft_list_builder_preimage(chains: Vec<Chain>) -> MmResult<SqlBuilder, Sql
     Ok(final_sql_builder)
 }
 
+fn nft_list_builder_preimage(table_name: &str, filters: Option<NftListFilters>) -> Result<SqlBuilder, SqlError> {
+    let mut sql_builder = SqlBuilder::select_from(table_name);
+    if let Some(filters) = filters {
+        if filters.exclude_spam {
+            sql_builder.and_where("possible_spam == 0");
+        }
+        if filters.exclude_phishing {
+            sql_builder.and_where("possible_phishing == 0");
+        }
+    }
+    drop_mutability!(sql_builder);
+    Ok(sql_builder)
```

Comment:
Do we apply `LIMIT` to this query after/before this function?

Reply:
Limit goes after, all filters are applied before `finalize_sql_builder` function.